tests/executables_src/testMapFileParser.cpp

```diff
@@ -341,4 +341,13 @@ BOOST_AUTO_TEST_CASE(testMapFileWithCommentsParse) {
 
 /*******************************************************************************************************************/
 
+BOOST_AUTO_TEST_CASE(testMapFileNewStyleMuxed) {
+  ChimeraTK::MapFileParser map_file_parser;
+  auto [regcat, mdcat] = map_file_parser.parse("newMuxedDataAcessor.mapp");
```

Comment:
This mapp file seems to be used exclusively in this particular (quite minimalistic) test case. Why can't you use newSequences.map instead?

Reply:
done and extended the test